rr/gstreamer/gst_media.py

```diff
@@ -91,6 +114,9 @@ def play_media(self):
         if gst.StateChangeReturn.SUCCESS != ret:
             raise GstMediaError("Unable to play the media")
 
+        # Send the buffer images to the installed callback
+        self.on_new_buffer()
```

Comment:
Will pipelines always have an `appsink`?

Reply:
This is not a good name. The `on_new_XXXX` is the name you would use for a callback that is to be called when a certain event occurs. Here you are just installing the callback. Please rename to something like `install_buffer_callback` or something similar.